### ISSUE TITLE
Formatted String

### ISSUE THREAD

**Issue by kolibs** [kolibs/eos#4]
Need a function to format a string with some constraints : 
* size : maximum size of a string
* string : string to format
* position : center / left / right

**Comment by xavitator** [comment]
Where is declared position? In the module (StrFormat?) or in an other module?

**Comment by kolibs** [comment]
I think in the module is a good idea...
we can define a type : 
```ocaml
 type position = Center
                | Left
                | Right 

    type fstring = {
         size : int ; 
         content : string ; 
         position : position 
    }
```

**Comment by xavitator** [comment]
Why do we need a type fstring ? Isn't it used only in the module ?

**Comment by kolibs** [comment]
You're right it's just a local type.

**Comment by xavitator** [comment]
The module is done, I wait the creation of the dev branch.

**Comment by kolibs** [comment]
Can you write a test in the tests/ repository ? You need to use this structure
```sh
tests/
|- name
    |- test.sh # Write your test in bash
```
This test need to return 1 in case of failure.

**Comment by xavitator** [comment]
For all tests, i think it's better to create a new issue.

**Comment by xavitator** [comment]
The issue is closed by the commit d139b878887ac57e5554f2e43051f5ca442c7dbf.
